bitnami/magento/templates/deployment.yaml

```diff
@@ -83,7 +83,7 @@ spec:
             - -ec
             - |
               mkdir -p "/bitnami/magento"
-              chown "/bitnami/magento"
+              chown -R {{ .Values.containerSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }} /bitnami/magento
```

Comment:
Why are you performing the operation recursively? The command below seems to do the same thing, but avoiding some other paths...

Reply:
I updated that line removing the '-R'